Header header

# величина магнитного поля по оси X [mGauss]
float32 magn_x

# величина магнитного поля по оси Y [mGauss]
float32 magn_y

# величина магнитного поля по оси Z [mGauss]
float32 magn_z
